modules/openy_gc_log/openy_gc_log.module

```diff
@@ -36,7 +36,9 @@ function openy_gc_log_preprocess_views_view(array &$variables) {
  * Implements hook_cron().
  */
 function openy_gc_log_cron() {
-  \Drupal::service("openy_gc_log.log_archiver")->archive();
+  $archiver = \Drupal::service("openy_gc_log.log_archiver");
+  $archiver->setWorkerChunkSize(600);
```

Comment:
Why `600`?

Reply:
I will change this to a setting, that can be modified in .settings.php